get-started/mining/index.md

```diff
@@ -78,8 +78,12 @@ meta_descr: meta_descr.mining
                     <h2>{% t mining.software %}</h2>
                         <p>{% t mining.softwarep %}
                         <ul>
-                            <li><a href="https://github.com/xmrig/xmrig">XMRig</a></li>
-                            <li><a href="https://github.com/cryptonote-social/csminer">CSminer</a></li>
+                            <li><a href="https://github.com/Cyrix126/gupaxx/">Gupaxx</a>
+                            <ul>
+                              <li><a href="https://xmrig.com/">XMRig</a></li>
+                              <li><a href="https://p2pool.io/#help">P2Pool</a></li>
+                            </ul>
+                            </li>
```

Comment:
Xmrig and p2pool should probably not be sub sections of gupaxx

Gupaxx includes xmrig and p2pool, but both xmrig and p2pool can be used standalone

The text can probably be updated too. You can p2pool mine from gui

```
There are several options when it comes to mining software. The CLI or GUI wallets can be used to control mining on a local Monero node. Alternatively, the GUI wallet may be used to mine on P2Pool.

If you want to mine to a traditional pool, or without running a node, you'll need additional software:
```